stanford_profile.profile

```diff
@@ -11,6 +11,35 @@ use Drupal\Core\Session\AccountInterface;
 use Drupal\Core\Field\FieldItemListInterface;
 use Drupal\Core\Access\AccessResult;
 
+/**
+ * Implements hook_install_tasks().
+ */
+function stanford_profile_install_tasks(&$install_state) {
+  return ['stanford_profile_final_task' => []];
+}
+
+/**
+ * Perform final tasks after the profile has completed installing.
+ *
+ * @param array $install_state
+ *   Current install state.
+ *
+ * @throws \Drupal\Component\Plugin\Exception\PluginException
+ */
+function stanford_profile_final_task(array &$install_state) {
+  /** @var \Drupal\stanford_profile\InstallTaskManager $install_task_manager */
+  $install_task_manager = \Drupal::service('plugin.manager.install_tasks');
+  foreach ($install_task_manager->getDefinitions() as $definition) {
+    /** @var \Drupal\stanford_profile\InstallTaskInterface $plugin */
+    $plugin = $install_task_manager->createInstance($definition['id']);
+    $plugin->runTask($install_state);
+  }
+
+  // We install some menu links, so we have to rebuild the router, to ensure the
+  // menu links are valid.
+  \Drupal::service('router.builder')->rebuildIfNeeded();
```

Comment:
It would be nice if these things were also tasks and ran in the final_task step.